pkg/apis/ceph.rook.io/v1/network_test.go

```diff
@@ -48,6 +48,24 @@ func TestNetworkCephIsHostLegacy(t *testing.T) {
 	assert.True(t, net.IsHost())
 }
 
+func TestNetworkCephIsHost(t *testing.T) {
```

Comment:
These tests should also include the default provider ("") as well as the legacy boolean in their matrix to make sure that they do the right thing related to other values.

And when things start to get more complicated, it can be good to use `t.Run("subtest name", ...` to define subtests that clearly spell out what is being tested.

Reply:
@BlaineEXE  wrote: 
> These tests should also include the default provider ("") as well as the legacy boolean in their matrix to make sure that they do the right thing related to other values.

I added it to the pre-existing test and will check for more possible places.

> 
> And when things start to get more complicated, it can be good to use `t.Run("subtest name", ...` to define subtests that clearly spell out what is being tested.

I will look around the code to understand yhe point of this better.